library/Faktory/Ent/Tracking.hs

```diff
@@ -0,0 +1,95 @@
+-- | Support for the @TRACK@ command (Enterprise only)
+--
+-- <https://github.com/contribsys/faktory/wiki/Ent-Tracking>
+--
+module Faktory.Ent.Tracking
```

Comment:
:clap: I really like reviewing Haskell like this. Some type definitions and one liner functions whose signatures tell me almost everything I need to know. Nice!